tests/src/helpers.c

```diff
@@ -22,6 +26,270 @@ static mbedtls_platform_context platform_ctx;
 
 mbedtls_test_info_t mbedtls_test_info;
```

Comment:
Now that we have setters/getters for this structure, can't we make this `static`?